### ISSUE TITLE
ledger: add sliders to adjust percentages for contributions

### ISSUE THREAD

**Issue by mrose17** [brave/browser-laptop#6873]
here is a fragment of the current about:preferences#payments page:
![0](https://cloud.githubusercontent.com/assets/400552/22344516/5b42b1ba-e3b1-11e6-8730-55b0a68ab41f.png)
add a slider to each publisher that auto-adjusts accordingly.

**Comment by NejcZdovc** [comment]
If I understand this issue correctly, it will be solved with #7347. There we will add input, where user can adjust percentage for each publisher.

**Comment by mrose17** [comment]
@NejcZdovc - i defer to @bsclifton on this point, he and i both thought that #7347 didn't have the actual sliders...

**Comment by NejcZdovc** [comment]
Yeah #7347 does not have any sliders, there is an input for a percentage. Will there be a slider in the table?

**Comment by mrose17** [comment]
yes. @bradleyrichter has an image (that he shared with me, but i can't find right now... oops!)

**Comment by bradleyrichter** [comment]
this is not final but the idea is to provide an adjustable linked slider for each site.

The slider should have an acceleration curve to provide more control in the first 50% range.

![image](https://cloud.githubusercontent.com/assets/13509546/23728118/f865253a-040f-11e7-9ead-5392f3d2cdd6.png)


This may be able to replace the exclude switch as well.

**Comment by NejcZdovc** [comment]
Ok, so  in the first phase I will do input and then when slider is confirmed we can add slider instead of input. Is this ok? @mrose17 @bradleyrichter

**Comment by mrose17** [comment]
fine by me.

**Comment by bradleyrichter** [comment]
fine with me as well
